check_source.py

```diff
@@ -11,6 +11,8 @@
 except ImportError:
     import cElementTree as ET
 
+from lxml import etree as ETL
```

Comment:
do we really need two XML libraries in here now? we have xml.etree(or cElementTree) and lxml both imported (works, of course, but…)

Reply:
actually cElementTree is supposed to die, but we can do that refactoring later IMO

Reply:
sure - no problem in going with what we have now